crates/toml_edit/src/de/value.rs

```diff
@@ -28,6 +28,9 @@ use crate::de::Error;
 /// assert_eq!(config.title, "TOML Example");
 /// assert_eq!(config.owner.name, "Lisa");
 /// ```
+///
+/// Already parsed [`crate::Value`] types can be turned into [`ValueDeserializer`]
+/// using [`serde::de::IntoDeserializer::into_deserializer`].
```

Comment:
nit: I feel like this should be before the `# Example` section.

At that point, it would probably be worthwhile to point out both relevant traits, something like
```md
To create a [`ValueDeserializer`], either use
- [`std::str::FromStr`]
- [`serde::de::IntoDeserializer::into_deserializer`] 
```